main.go

```diff
@@ -67,9 +68,10 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
@danielvegamyhre this is one issue I am unsure on.  

I am not sure what 9443 corresponds too and what that option is now.

/hold

Reply:
The port was for the webhook server.  

The metricsAddr that you have `:8080` from the flag is being set on the now new `server.Options{BindAddress: ":8080"}`

The webhook has it's own options now set using WebhookServer

Reply:
Thanks for helping on that one.  Added this.